pkgs/development/libraries/softhsm/default.nix

```diff
@@ -0,0 +1,30 @@
+{ stdenv, fetchurl, botan }:
+
+stdenv.mkDerivation rec {
+
+  name = "softhsm-${version}";
+  majorVersion = "2";
+  version = "${majorVersion}.0.0";
+
+  src = fetchurl {
+    url = "https://dist.opendnssec.org/source/${name}.tar.gz";
+    sha256 = "eae8065f6c472af24f4c056d6728edda0fd34306f41a818697f765a6a662338d";
+  };
+
+  configureFlags = [
+    "--with-crypto-backend=botan"
+    "--with-botan=${botan}"
+    "--sysconfdir=$out/etc"
+    "--localstatedir=$out/var"
```

Comment:
Does it expect to write things to $out/var ... because that would fail since it's not possible to write to the store. I'm not sure what the correct path would be, is there a suitable default?